src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -475,6 +475,7 @@ def test_static_theme_wizard_listed(self):
         doc = pq(response.content)
         assert doc('#theme-wizard')
         assert doc('#theme-wizard').attr('data-version') == '1.0'
+        assert doc('#theme-name').attr('type') == 'text'
```

Comment:
To make this test more explicit and actually easier to read we should use `input#theme-name` instead. This way it's better to see what you can expect from `attr`

Reply:
https://github.com/mozilla/addons-server/pull/8798/commits/ca8f9d36800573e7158a30d889b65ce7e954606c